linux/mapboxgl-app.gypi

```diff
@@ -39,6 +39,10 @@
         ],
       },
 
+      'include_dirs': [
+        '../src',
```

Comment:
What's this necessary for? IIRC, we want to use the apps partly as as examples, which means they should avoid using internal includes.